### PR TITLE
IW-1470 | watch show module

### PR DESCRIPTION
**Pull request by latata**
## Links

* http://wikia-inc.atlassian.net/browse/IW-1470

## Description

Shows watch show button below the first portable infobox if wgWatchShowURL is set.

## Reviewers

@Wikia/iwing

### CODE REVIEW

File: app/components/watch-show.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  {
+    wikiVariables: service(),
+    geo: service(),
+
+    tagName: '',
+
+    url: oneWay('wikiVariables.watchShowURL'),
+
+    isVisible: computed('url', function () {
+      return this.url && this.geo.country === 'PL';
```
Comment (xkxd) [inline app/components/watch-show.js]: for testing purposes only? 🙈

Comment (latata) [reply app/components/watch-show.js]: oops. good catch!

File: app/templates/components/watch-show.hbs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{{#if isVisible}}
+  <div class="watch-show">
+    <a class="wds-button" onclick={{this.trackClick}} href={{url}}>Watch now</a>
+    {{! todo fix the hulu asset url }}
```
Comment (xkxd) [inline app/templates/components/watch-show.hbs]: when will this 'todo' happen?

**Review by xkxd — APPROVED** [review]

File: app/templates/components/watch-show.hbs (hunk 2 above)
Comment (latata) [reply app/templates/components/watch-show.hbs]: when I get the asset from Derek :)

**Review by bkoval — APPROVED** [review]

**Pull request merged**